### ISSUE TITLE
MinSdkVersion

### ISSUE THREAD

**Issue by arjava** [hieupham1993/android-standalone-scroll-bar#2]
why is this only available in minSdkVersion 21 huh?
I'm using the minSdkVersion 19 project, can it not be changed to 19 or is there a dependency related to minSdkVersion 21 that must use this?

**Comment by hieupham1993** [comment]
Yeah. This lib using tint api which avalable on minSdkVersion 21. But I think it's not a big deal. You can import this lib to your project as sub module and modify some lines of code to achieve what you desire.

1. Change minSdkVersion to 19
2. In StandaloneScrollBar.kt
Update trackView and thumbView implementation to some View that implement TintableBackgroundView. Eg: AppCompatImageView
```kotlin
Line 55: internal val trackView: View by lazy { AppCompatImageView(context) }
Line 57: internal val thumbView: View by lazy { AppCompatImageView(context) }
Line 128: ViewCompat.setBackgroundTintList(trackView, value)
Line 137: ViewCompat.setBackgroundTintList(thumbView, value)
```
3. In OrientationHelper.kt
```kotlin
Line 38: ViewCompat.setBackgroundTintList(trackView, it)
Line 46: ViewCompat.setBackgroundTintList(thumbView, it)
```

Hope this help!
